doc/sphinx/changes.rst

```diff
@@ -14,7 +14,112 @@ Version 8.19
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
-TODO
+Coq version 8.19 extends the kernel universe polymorphism to
+polymorphism over sorts (e.g. `Prop`, `SProp`) along with a few new
+features, a host of improvements to the notation system, the Ltac2
+standard library, and the removal of some standard library files after
+a long deprecation period.
+
+We highlight some of the most impactful changes here:
+
+  - :ref:`sort-polymorphism` makes it possible to share common constructs
+    over `Type` `Prop` and `SProp`.
+
+  - The notation :g:`term%_scope` to set a scope only temporarily
+    (in addition to :g:`term%scope` for opening a
+    scope applying to all subterms).
+
+  - :tacn:`lazy`, :tacn:`simpl`, :tacn:`cbn` and :tacn:`cbv` and the associated :cmd:`Eval`
+    and :tacn:`eval` reductions learned to do head reduction when given flag `head`.
+
+  - :ref:`New Ltac2 APIs <819Ltac2>`, improved Ltac2 `exact` and
+    dynamic building of Ltac2 term patterns.
+
+  - New performance evaluation facilities: :cmd:`Instructions` to
+    count CPU instructions used by a command (Linux only) and
+    :ref:`profiling` system to produce trace files.
+
+  - New command :cmd:`Attributes` to assign attributes such as
+    :attr:`deprecated` to a library file.
+
+Notable breaking changes:
+
+  - :tacn:`replace` with `by tac` does not automatically attempt to solve
+    the generated equality subgoal using the hypotheses.
+    Use `by first [assumption | symmetry;assumption | tac]`
+    if you need the previous behaviour.
+
+  - :ref:`Removed old deprecated files <819Stdlib>` from the standard library.
+
+See the `Changes in 8.19.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.19 <https://coq.github.io/doc/v8.19/refman>`_,
+`documentation of the 8.19 standard library <https://coq.github.io/doc/v8.19/stdlib>`_
+and `developer documentation of the 8.19 ML API <https://coq.github.io/doc/v8.19/api>`_
+are also available.
+
+Maxime Dénès and Thierry Martinez moved the CI away from `gitlab.com <http://gitlab.com>` to
+use Inria supported runner machines through  `gitlab.inria.fr <https://gitlab.inria.fr>`.
+
+Théo Zimmermann maintained `coqbot <https://github.com/coq/bot>` used
+to run Coq's CI and other pull request management tasks.
+
+Jason Gross maintained the `bug minimizer <https://github.com/JasonGross/coq-tools>`
+and its `automatic use through coqbot <https://github.com/coq/coq/wiki/Coqbot-minimize-feature>`.
+
+Jaime Arias and Erik Martin-Dorel have maintained the `Coq Docker images
+<https://hub.docker.com/r/coqorg/coq>`_ that are used in many Coq
+projects for continuous integration.
+
+Maxime Dénès, Paolo G. Giarrusso, Huỳnh Trần Khanh, Romain Tetley and Laurent Théry have
+maintained the VsCoq extension for VS Code.
+
```

Comment:
What's the criteria for mentioning projects in this list? Should be Dune, jsCoq, SerAPI, coq-lsp mentioned?

Reply:
Not sure TBH
I think we want work which supports the development of Coq, not work aimed at users (that would go more in coq platform credits or individual project credits)
by that logic I think we would include dune and remove vscoq

Reply:
From that perspective, the connection between the Platform and VsCoq is very weak. From my perspective, the inclusion of VsCoq makes sense due to it being a long-term Coq Consortium project (like what the Coq Platform now is becoming, in effect).

Reply:
@ejgallego not sure which projects will end up included but which people would you mention for the projects you suggested?

Reply:
Off the top of my head (I can verify this later)

- jsCoq is Shachar Itzhaky and me (plus some contributors)
- Dune is Ali Caglayan, Rudi Gringerb, and me (and actually Rodolphe Lepigre did important contribs)
- coq-lsp is Ali, Bhakti Shah, me (and maybe some other important contribs should be mention)
- serApi is me, Karl, and Coq team (due to many overlays)

Reply:
some discussion is happening on zulip
https://coq.zulipchat.com/#narrow/stream/237656-Coq-devs-.26-plugin-devs/topic/credits.20for.208.2E19